docs/docsite/rst/scenario_guides/guide_aci.rst

```diff
@@ -543,13 +543,13 @@ Here is a small overview of useful operational tasks to reuse in your playbooks.
 Feel free to contribute more useful snippets.
 
 
-Waiting for all controllers to be ready
-.......................................
-You can use the below task after you started to build your APICs and configured the cluster to wait until all the APICs have come online. It will wait until the number of controllers equals the number listed in the ``apic`` inventory group.
+Waiting for all control nodes to be ready
+.........................................
+You can use the below task after you started to build your APICs and configured the cluster to wait until all the APICs have come online. It will wait until the number of control nodes equals the number listed in the ``apic`` inventory group.
```

Comment:
```suggestion
You can use the below task after you started to build your APICs and configured the cluster to wait until all the APICs have come online. It will wait until the number of controllers equals the number listed in the ``apic`` inventory group.
```